docs/en/observability/synthetics-troubleshooting.asciidoc

```diff
@@ -0,0 +1,81 @@
+[[synthetics-troubleshooting]]
+= Troubleshooting Synthetics
+
+++++
+<titleabbrev>Troubleshooting</titleabbrev>
+++++
+
+[discrete]
+[[synthetics-troubleshooting-local-debugging]]
+= Local debugging
+
+For debugging synthetic tests locally, you can set an environment variable,
```

Comment:
nit: For debugging synthetic tests locally, set `DEBUG=synthetics` environment variable to capture Synthetics agent logs.

running tests locally repeated 2 times.